doc/xlator.md

```diff
@@ -22,7 +22,7 @@ volume.auth.password
 ```
 brick.id
 brick.hostname
-brick.nodeid
+brick.peerid
```

Comment:
This change is a problem if glusterfs xlator code is not changed to use `peerid`. (I just noticed `$SRC/glusterd2/brick/types.go` is already modified, we need to change the glusterfs xlator option table if `{{ brick.nodeid }}` is used